boring/src/ssl/mod.rs

```diff
@@ -2826,6 +2845,20 @@ impl SslRef {
         }
     }
 
+    /// Sets the ongoing session's supported groups by their named identifiers
+    /// (formerly referred to as curves).
+    #[corresponds(SSL_set1_groups)]
+    pub fn set_group_nids(&mut self, group_nids: &[i32]) -> Result<(), ErrorStack> {
```

Comment:
`&[i32]` should use the newtype (BTW, `i32` isn't guaranteed to be `c_int`)

If you use `#[repr(transparent)]` on the newtype, then it will be safe to cast the pointer to the slice, without copying.